kani-driver/src/call_cargo.rs

```diff
@@ -80,10 +82,18 @@ impl KaniSession {
             });
         }
 
+        let mut symtabs = glob(&outdir.join("*.symtab.json"))?;
+        let mut metadata = glob(&outdir.join("*.kani-metadata.json"))?;
+        if std::env::var("IS_KANI_PROPTEST").is_ok() {
```

Comment:
Why is this proptest specific? It seems like we're adding extern libs, of which proptest is only one possible example

Reply:
Turns out not doing this triggers a bug documented in #1478.

I'm OK with not merging this until #1478 is fixed. Otherwise, this is the only way to pass CI.

Reply:
I posted a comment in the issue you created. Can you please check?